analyzers/tests/SonarAnalyzer.UnitTest/Common/RuleTest.cs

```diff
@@ -113,6 +116,51 @@ public void SonarDiagnosticAnalyzer_IsUsedInAllRules()
             }
         }
 
+        [TestMethod]
+        public void Verify_ConcurrentProcessingDisabledByDefault()
+        {
+            var retriever = new ConcurrentProcessingRetriever();
+            retriever.IsConcurrentProcessingDisabled.Should().BeNull();
+            Verifier.VerifyAnalyzer(new[] { "TestCasesForRuleFailure\\SpecialCases.cs" }, retriever);
```

Comment:
I think that in all tests we can use VerifyNoExceptionThrown instead of VeriFyAnalyzer. The intention is to see that the rule is initialized and no exceptions are thrown during analysis.